tests/unit_test/data_processor/data_processor_test.py

```
@@ -1638,11 +1638,80 @@ def test_resolve_image_link_no_image(self):
+
+        assert POSIntegrations.objects(bot=bot, provider=provider, sync_type=sync_type).count() == 1
+
+        result = CognitionDataProcessor.delete_pos_integration_config(bot, provider, sync_type)
+
+        assert result == {"provider": provider, "sync_type": sync_type}
+        assert POSIntegrations.objects(bot=bot, provider=provider, sync_type=sync_type).count() == 0
+        POSIntegrations.objects(provider= "petpooja").delete()
+
+    def test_delete_pos_integration_config_failure(self):
+        bot = "test_bot"
+        user = "test_user"
```

Ruff (0.11.9)

Local variable `user` is assigned to but never used

Remove assignment to unused variable `user`

(F841)